template.json

```diff
@@ -7,7 +7,9 @@
     "/dns4/p2p.canarytest-2.kusama.network/tcp/30100/p2p/QmYG1YUekKETmD68yFKbjXDRbSAFULRRJpb1SbQPuSKA87",
     "/dns4/p2p.canarytest-3.kusama.network/tcp/30100/p2p/QmQv5EXUAfVt4gbupiuLDZP2Gd7ykK6YuXoYPkyLfLtJch",
     "/dns4/p2p.canarytest-4.kusama.network/tcp/30100/p2p/QmP3zYRhAxxw4fDf6Vq5agM8AZt1m2nKpPAEDmyEHPK5go",
-    "/dns4/p2p.canarytest-5.kusama.network/tcp/30100/p2p/QmdePe9MiAJT4yHT2tEwmazCsckAZb19uaoSUgRDffPq3G"
+    "/dns4/p2p.canarytest-5.kusama.network/tcp/30100/p2p/QmdePe9MiAJT4yHT2tEwmazCsckAZb19uaoSUgRDffPq3G",
+    "/ip4/kusama-bootnode-0.paritytech.net/tcp/30333/p2p/QmaEYixpsZy126nijCeP1LFhUipvGs23RU15jJJyAePxW",
```

Comment:
I think the protocol name should be `dns4` instead of `ip4` if you want to use a dns name. Also, why is this peer ID one character shorter?